tst/standard/grahom.tst

```diff
@@ -2498,6 +2498,211 @@ Error, the 1st argument (a digraph) must not satisfy IsMultiDigraph
+gap> LatticeDigraphEmbedding(D, G);
+Transformation( [ 4, 3, 2, 1 ] )
+gap> D := Digraph([[2], [3], [4], []]);
+<immutable digraph with 4 vertices, 3 edges>
+gap> G := Digraph([[2], [3], [4], [5], []]);
+<immutable digraph with 5 vertices, 4 edges>
+gap> D := DigraphReflexiveTransitiveClosure(D);
+<immutable preorder digraph with 4 vertices, 10 edges>
+gap> G := DigraphReflexiveTransitiveClosure(G);
+<immutable preorder digraph with 5 vertices, 15 edges>
+gap> LatticeDigraphEmbedding(G, D);
+fail
+gap> LatticeDigraphEmbedding(D, G);
+IdentityTransformation
+gap> D := Digraph([[2, 3, 5], [4, 6], [4, 7, 9], [8, 11], [6, 10], [11],
+>   [8, 12], [14], [10, 13], [11, 12], [14], [14], [14], []]);
+<immutable digraph with 14 vertices, 23 edges>
+gap> N5 := Digraph([[2, 4], [3], [5], [5], []]);
+<immutable digraph with 5 vertices, 5 edges>
+gap> D := DigraphReflexiveTransitiveClosure(D);
+<immutable preorder digraph with 14 vertices, 66 edges>
+gap> N5 := DigraphReflexiveTransitiveClosure(N5);
+<immutable preorder digraph with 5 vertices, 13 edges>
+gap> LatticeDigraphEmbedding(N5, D);
+Transformation( [ 1, 9, 10, 2, 11, 6, 7, 8, 9, 10, 11 ] )
+gap> D := Digraph([[2], [3], [4], []]);
+<immutable digraph with 4 vertices, 3 edges>
+gap> G := Digraph([[2, 3], [4], [4], []]);
+<immutable digraph with 4 vertices, 4 edges>
+gap> D := DigraphReflexiveTransitiveClosure(D);
+<immutable preorder digraph with 4 vertices, 10 edges>
+gap> G := DigraphReflexiveTransitiveClosure(G);
+<immutable preorder digraph with 4 vertices, 9 edges>
+gap> LatticeDigraphEmbedding(D, G);
+fail
+gap> LatticeDigraphEmbedding(G, D);
+fail
+gap> N5 := Digraph([[2, 4], [3], [5], [5], []]);
+<immutable digraph with 5 vertices, 5 edges>
+gap> H := Digraph([[], [1], [1], [2], [3], [2, 3], [4, 5, 6]]);
+<immutable digraph with 7 vertices, 9 edges>
+gap> N5 := DigraphReflexiveTransitiveClosure(N5);
+<immutable preorder digraph with 5 vertices, 13 edges>
+gap> H := DigraphReflexiveTransitiveClosure(H);
+<immutable preorder digraph with 7 vertices, 22 edges>
+gap> LatticeDigraphEmbedding(N5, H);
+Transformation( [ 7, 5, 3, 4, 1, 6, 7 ] )
+gap> D := Digraph([[2, 6], [3, 7], [4], [], [4], [5, 7], [4]]);
+<immutable digraph with 7 vertices, 9 edges>
+gap> G := DigraphRemoveVertex(D, 7);
+<immutable digraph with 6 vertices, 6 edges>
+gap> G := DigraphReflexiveTransitiveClosure(G);
+<immutable preorder digraph with 6 vertices, 17 edges>
+gap> D := DigraphReflexiveTransitiveClosure(D);
+<immutable preorder digraph with 7 vertices, 22 edges>
+gap> DigraphEmbedding(G, D);
+IdentityTransformation
+gap> LatticeDigraphEmbedding(G, D);
+fail
+gap> D := CycleDigraph(5);
+<immutable cycle digraph with 5 vertices>
+gap> G := Digraph([[1]]);
+<immutable digraph with 1 vertex, 1 edge>
+gap> LatticeDigraphEmbedding(D, G);
+Error, the 1st argument (a digraph) must be a lattice digraph
+gap> LatticeDigraphEmbedding(G, D);
+Error, the 2nd argument (a digraph) must be a lattice digraph
```

Comment:
It would be good to have a few more examples on larger lattices - you could test them with `IsDigraphEmbedding`.